pkg/imgpkg/cmd/label_flags.go

```diff
@@ -0,0 +1,16 @@
+// Copyright 2020 VMware, Inc.
+// SPDX-License-Identifier: Apache-2.0
+
+package cmd
+
+import (
+	"github.com/spf13/cobra"
+)
+
+type LabelFlags struct {
+	Labels map[string]string
+}
+
+func (l *LabelFlags) Set(cmd *cobra.Command) {
+	cmd.Flags().StringToStringVarP(&l.Labels, "labels", "l", make(map[string]string), "Set labels on image")
```

Comment:
How much more complicated do you think it would be if we allowed the following
`imgpkg push -f folder -i registry.io/my-img -l my.label=10 -l my.otherlabel="something interesting" -l yet.another.label="this is the real deal"`?
I think we might have to make this a `StringSliceVarP` and we would have to process each value to split by `=`.

Reply:
I think that works as-is right now, or maybe I'm not understanding what you're suggesting.

```shell
$ ./imgpkg push -i jmsearcy/test-imgpkg-image -f ./pkg/imgpkg/cmd/test_assets/image_with_config -l my.label=10 -l my.otherlabel="something interesting" -l yet.another.label="this is the real deal"

dir: .
file: config.yml
Pushed 'index.docker.io/jmsearcy/test-imgpkg-image@sha256:acafc2d67e1fa9093d08c75399b5e60f0e60851741bb3e604195db19b3a09a40'
Succeeded
```

```shell
$ regctl image inspect jmsearcy/test-imgpkg-image

{
  "created": "0001-01-01T00:00:00Z",
  "architecture": "",
  "os": "",
  "config": {
    "Labels": {
      "my.label": "10",
      "my.otherlabel": "something interesting",
      "yet.another.label": "this is the real deal"
    }
  },
  "rootfs": {
    "type": "layers",
    "diff_ids": [
      "sha256:284871f9fdadd60646be5ee42a95afa16c9352ef41ef5c1490dce02aa08bb338"
    ]
  },
  "history": [
    {
      "created": "0001-01-01T00:00:00Z",
      "created_by": "imgpkg",
      "author": "imgpkg"
    }
  ]
}
```

Reply:
My bad. I assume that you needed a SliceVar to be able to use multiple flags